packages/entities/entities-plugins/src/components/free-form/Datakit/flow-editor/composables/useNodeForm.ts

```diff
@@ -280,3 +284,31 @@ export function useNodeForm<T extends BaseFormData = BaseFormData>(
     nameValidator: useNameValidator(),
   }
 }
+
+export function useSubSchema(subSchemaName: Exclude<NodeType, 'request' | 'service_response'>) {
+  const { getSchema } = useFormShared()
+  return computed(() => {
+
+    // Implicit nodes do not have schema, we hardcoded schemas for them
+    if (isImplicitType(subSchemaName)) {
+      switch (subSchemaName) {
+        case 'response':
+          return ResponseSchema
+        case 'service_request':
+          return ServiceRequestSchema
+      }
```

Comment:
Do we need the same for the `request` and `service_response` nodes?

Reply:
No, they don't have form